analysis_options.yaml

```diff
@@ -6,7 +6,7 @@ analyzer:
 linter:
   rules:
     # Make constructors the first thing in every class
-    sort_constructors_first: false
+    sort_constructors_first: true
```

Comment:
This change is 100% correct and I would like to appreciate your effort for understanding the rules. Another tip for you is that if we don't specify rule in this file, it is automatically set to true. It means, we can even remove this entire line and rule will still be enabled.

Reply:
@M-A-D-A-R-A So it would be better if you remove this line

Reply:
@M-A-D-A-R-A So it would be better if you remove this line